Source/NSData.m

```diff
@@ -253,6 +253,35 @@
       NSWarnFLog(@"Open (%@) attempt failed - bad path", path);
       return NO;
     }
+  
+#ifdef __ANDROID__
+  // Android: try using asset manager if path is in main bundle resources
+  AAsset *asset = [NSBundle assetForPath:path];
```

Comment:
As all the following operations are already present at GSFileHandle wouldn't it be better to reuse these?

Reply:
We probably could, but it would have two downsides:

- As NS/GSFileHandle doesn’t have a way to pass in an AAsset object, we’d have to open the AAsset twice: once to check if we should take that code path (as currently done), and then once more in GSFileHandle.
- Also, we now use [`AASSET_MODE_BUFFER`](https://developer.android.com/ndk/reference/group/asset#group___asset_1gga16af7b253440dadd46a80a4b9fddba4da40ec098f4afb7c2869fa449d3059f6bb) in NSData, whereas GSFileHandle needs to use `AASSET_MODE_RANDOM` to support random access, so it might be slightly slower to use GSFileHandle (although I have not profiled this).

Let me know if you still think it’d be better to use GSFileHandle here.